### ISSUE TITLE
new_delete - Wsized-deallocation

### ISSUE THREAD

**Issue by Wiz-IO** [raspberrypi/pico-sdk#304]
pico_standard_link\new_delete.cpp: warning: the program should also define 'void operator delete [](void*, unsigned int)' [-Wsized-deallocation]

workaround
```cpp
#pragma GCC diagnostic push
#pragma GCC diagnostic ignored "-Wsized-deallocation"
// the code
#pragma GCC diagnostic pop
```

**Comment by kilograham** [comment]
compiler? version?

**Comment by Wiz-IO** [comment]
gcc version 7.2.1 20170904 (release) 
[ARM/embedded-7-branch revision 255204] (GNU Tools for Arm Embedded Processors 7-2017-q4-major)

**Comment by Wiz-IO** [comment]
4 delete - no warn
https://github.com/Wiz-IO/framework-wizio-pico/blob/main/SDK/pico/pico_standard_link/new_delete.cpp#L22

**Comment by lurch** [comment]
Fixed by #430 ?

**Comment by kilograham** [comment]
merged into develop
